roles/ocluster/tasks/main.yml

```
@@ -0,0 +1,41 @@
+---
+
+- name: Download the source from GitHub
+  git:
+    repo: https://github.com/ocurrent/ocluster.git
+    dest: "{{ ansible_env.HOME }}/ocluster"
+    force: yes
+
+- name: Download the source from GitHub
+  shell: git fetch origin refs/pull/193/head && git reset --hard 104b505
+  args:
+    chdir: "{{ ansible_env.HOME }}/ocluster"
+
+- name: Add remote obuilder
+  shell: git remote add misterda https://github.com/MisterDA/obuilder.git && git fetch --all
+  args:
+    chdir: "{{ ansible_env.HOME }}/ocluster/obuilder"
+
+- name: Checkout the correct SHA
+  shell: git checkout 20229cbdae4dc55b3f1ca5e68c6c48e337dd91d7
+  args:
+    chdir: "{{ ansible_env.HOME }}/ocluster/obuilder"
+
+- name: Update Fetcher variable to User_temp
+  replace:
+    path: "{{ ansible_env.HOME }}/ocluster/worker/obuilder_build.ml"
+    regexp: '^module Fetcher = Obuilder.Docker$'
+    replace: 'module Fetcher = Obuilder.User_temp'
```

```suggestion
- name: Checkout the correct SHA
  shell: git fetch origin master && git checkout 376d1fea0d832f9b0b1c947a0fc569cb870d3910
  args: 
    chdir: "{{ ansible_env.HOME }}/ocluster"
```

Switched to running master version on mac-mon-1 using this configuration.

---

I have reinstated the edit to `ocluster/worker/obuilder_build.ml` to set `module Fetcher` as this is still needed.